sgkit/stats/genedrop_numba_fns.py

```diff
@@ -0,0 +1,169 @@
+import numpy as np
+
+from sgkit.accelerate import numba_guvectorize, numba_jit
+from sgkit.typing import ArrayLike
+
+from .pedigree import _compress_hamilton_kerr_parameters, topological_argsort
+
```

Comment:
Maybe add a technical note about randomness here just so the reader can understand what np.random is doing below?

Reply:
Yes, forgot that sorry.